ministryofjustice/correspondence_tool_staff
# CT-2266 Close offender SAR case

## Description
Use existing pattern for closing a case instead of "Mark as closed":


## Self-review checklist

 
### Screenshots
![image](https://user-images.githubusercontent.com/1161161/62821421-8bedf280-bb6c-11e9-9f2e-23f92a568e46.png)

![image](https://user-images.githubusercontent.com/1161161/62821423-90b2a680-bb6c-11e9-9ebc-ebe8ac89402e.png)

  
### Related JIRA tickets
https://dsdmoj.atlassian.net/browse/CT-2266
 

### Manual testing instructions
This builds on the previous work to provide manual buttons for stepping a case through the workflow states all the way to closed. To test the new feature:

1. Create a new Offender SAR case as Brian Rix

2. Take it through to "ready to dispatch" 

3. The action button should say "Close"

4. Closing should be a case of clicking close; supply the date of dispatch and then the outcome category. 

5. Case should no longer appear in Open Cases

6. Case should now appear in Closed Cases and the case history should reflect the change 

7. The time taken should be calculated and displayed on the Case page and the outcome reason should be displayed

![image](https://user-images.githubusercontent.com/1161161/63949680-ad0e7880-ca72-11e9-8461-fa302d3453e1.png)
